src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -59,12 +67,35 @@ private static object RegExSetup(PowerFxConfig config)
             return null;
         }
 
+        private static void AddResourceManager(RuntimeConfig runtimeConfig)
+        {
+            runtimeConfig.AddService<IResourceManager>(new DefaultResourceManager());
+        }
+
+        private static object BlobSetup(PowerFxConfig config, SymbolTable symbolTable)
+        {
+            config.AddBlobTestFunctions();
+            config.EnableSetFunction();
+
+            return new List<ISymbolSlot>()
+            {
+                symbolTable.AddVariable("blob", FormulaType.Blob, true, "blob"),
+                symbolTable.AddVariable("str", FormulaType.String, true, "str")
+            };
+        }
+
+        private static void AddBlobVar(ReadOnlySymbolValues symbolValues, ISymbolSlot[] slots)
+        {
+            symbolValues.Set(slots[0], FormulaValue.NewBlank(FormulaType.Blob));
+            symbolValues.Set(slots[1], FormulaValue.NewBlank(FormulaType.String));
+        }

```

Comment:
This is very loosely coupled with the function above - we need to know that there are 2 items in the slots... Can we pass the result of updatePfxConfig to this method, and it will cast to ISymbolSlot[] inside here?

Reply:
I fixed it